README.md

```diff
@@ -55,6 +55,26 @@ This is the place to:
 Note: there is no need to repeat everything that is already well-documented in
 the code.
 
+If you would like to include math equations to explain parts of your code or repo,
+it is recommended to use an external renderer and insert prepared svg files here.
+For example, it's possible to insert a link to the online CodeCogs equation editor;
```

Comment:
We've used MathJax previously, I'm not familiar with CodeCogs -- @silviacorbellaro

Reply:
My understanding is that MathJax is less supported in GitHub rendering, but I am not really familiar with them. Any url renderer would be acceptable and the preference is just for any that can parse non-percent encoded symbols for readability. Potentially relevant exchange on SO:

https://stackoverflow.com/questions/11256433/how-to-show-math-equations-in-general-githubs-markdownnot-githubs-blog

Reply:
Folks, does D-Wave already have a license for CodeCogs? I took a brief look and there are a few options to buy a license to the tool. I can't visualize the license text without signing up. What I can see is that there is a separate licence for commercial use, it's just more expensive.